test/test/test_hash.c

```diff
@@ -1168,6 +1168,8 @@ static int test_hash_iteration(void)
 	handle = rte_hash_create(&ut_params);
 	RETURN_IF_ERROR(handle == NULL, "hash creation failed");
 
+	rte_hash_iterator_init(handle, &state);
```

Comment:
Replace the line above with:
`RETURN_IF_ERROR(rte_hash_iterator_init(handle, &state) != 0, "initialization of the hash iterator failed");`